src/executor/package.json

```diff
@@ -11,7 +11,8 @@
     "build": "tsc",
     "test": "jest",
     "testOpenHandles": "jest --detectOpenHandles",
-    "test-watch": "jest --watch"
+    "test-watch": "jest --watch",
+    "start-staging": "tsc && API_BASE_URL=\"https://api-staging.flooq.io\" node ./dist/source/Index.js"
```

Comment:
I wouldn't do it with an npm script. You can add a `.env` file or a `.env.local` file and set the variable there. This will be required anyway once we add the authentication.

Reply:
I get your point. But at that point we'll have to rework quite a few things and I'd extract it just then.

Reply:
Not sure what you mean. It's possible to to it right now without any adjustment except changing the API_BASE_URL in the .env file.

Reply:
![image](https://user-images.githubusercontent.com/7260168/166068676-709c92fe-c797-4d07-96a9-29bc51357337.png)
Just change the url here.
And if you want to make sure that you don't commit this change you can add a `.env.local` file which automatically overrides settings in the `.env` file.

Reply:
My initial thinking was that I didn't want this to be the default but an option available to all devs. That's why I added it as a `npm run`option instead of changing the `.env` file or just creating a `.env.local` file. Do you know of a way to specify the env file for a `node` command?

I'll talk to the other executor devs if this should be the default though, as it is a lot easier to work with because you don't need to run the other containers locally and don't have to have the setup to be able to do so.

Reply:
I think we should discuss this in person because we talk about different things

Reply:
Btw if you add a .env.local it won't be added to git because it is ignored. The idea is that these are your own settings.

Reply:
Documentation for future reference: node itself (`dotenv` to be precise) doesn't support `.env.local` files.